app/controllers/holidays_controller.rb

```diff
@@ -0,0 +1,18 @@
+class HolidaysController < ApplicationController
+  before_action :authorise_for_managing_resources!
+
+  def index
+    respond_to do |format|
+      format.html
+      format.json do
+        render json: Holiday.merged_for_calendar_view
```

Comment:
Just spoke about this. Answer is no, because we render all the events in one go.